discord/ext/commands/core.py

```diff
@@ -143,10 +144,10 @@ def resolve_annotation(annotation: Any, globalns: Dict[str, Any], cache: Dict[st
         annotation = ForwardRef(annotation)
     return _evaluate_annotation(annotation, globalns, cache)
 
-def get_signature_parameters(function: types.FunctionType) -> Dict[str, inspect.Parameter]:
+def get_signature_parameters(function: types.FunctionType) -> collections.OrderedDict[str, inspect.Parameter]:
```

Comment:
The return type hint used here is Python 3.9 syntax. Python 3.8 only has the generic for `OrderedDict` in `typing.OrderedDict`